internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/WriteComputeWithActiveActiveReplicationTest.java

```diff
@@ -802,30 +813,30 @@ public void testAAReplicationForPartialUpdateOnMapField() throws IOException {
     verifyFLMRecord(storeName, dc1RouterUrl, key2, regularFieldValue, arrayFieldValue, expectedMapFieldValue);
     verifyFLMRecord(storeName, dc0RouterUrl, key2, regularFieldValue, arrayFieldValue, expectedMapFieldValue);
 
-    // TODO: When PartialPut with the same timestamp as PUT is processed, it wipes out the data with the
-    // timestamp <= t2 including PUTs timestamp. This creates a race between PartialPut and PUT
-    /*
     ub = new UpdateBuilderImpl(wcSchemaV1);
     mapFieldValue = new HashMap<>();
     mapFieldValue.put("six", 6);
     mapFieldValue.put("seven", 7);
     ub.setNewFieldValue(MAP_FIELD, mapFieldValue);
     timestampedOp = new VeniceObjectWithTimestamp(ub.build(), 2);
     sendStreamingRecord(systemProducerMap.get(childDatacenters.get(0)), storeName, key2, timestampedOp);
+    expectedMapFieldValue.putAll(mapFieldValue);
+    expectedMapFieldValue.remove("two");
```

Comment:
`two` and `three` were added by `Full Put`. How come `Partial Put` removes them?

Reply:
There was a discussion in the write path channel, we talked about in the field level TS world, full PUT and partial PUT on a same field should have the same priority. So that's to say, for collection field, we will (for now use colo ID to resolve it, and in the future compare collection value to resolve it)